google-cloud-storage/src/main/java/com/google/cloud/storage/multipartupload/model/ListMultipartUploadsRequest.java

```diff
@@ -0,0 +1,305 @@
+  @BetaApi
+  public String delimiter() {
+    return delimiter;
+  }
+
+  /**
+   * The encoding type used by Cloud Storage to encode object names in the response.
+   *
+   * @return The encoding type.
+   * @since 2.61.0 This new api is in preview and is subject to breaking changes.
+   */
+  @BetaApi
+  public String encodingType() {
+    return encodingType;
+  }
+
+  /**
+   * Together with {@code upload-id-marker}, specifies the multipart upload after which listing
+   * should begin.
+   *
+   * @return The key marker.
+   * @since 2.61.0 This new api is in preview and is subject to breaking changes.
+   */
+  @BetaApi
+  public String keyMarker() {
+    return keyMarker;
+  }
+
+  /**
+   * The maximum number of multipart uploads to return.
+   *
+   * @return The maximum number of uploads.
+   * @since 2.61.0 This new api is in preview and is subject to breaking changes.
+   */
+  @BetaApi
+  public Integer maxUploads() {
+    return maxUploads;
+  }
+
+  /**
+   * Filters results to multipart uploads whose keys begin with this prefix.
+   *
+   * @return The prefix.
+   * @since 2.61.0 This new api is in preview and is subject to breaking changes.
+   */
+  @BetaApi
+  public String prefix() {
+    return prefix;
+  }
+
+  /**
+   * Together with {@code key-marker}, specifies the multipart upload after which listing should
+   * begin.
+   *
+   * @return The upload ID marker.
+   * @since 2.61.0 This new api is in preview and is subject to breaking changes.
+   */
+  @BetaApi
+  public String uploadIdMarker() {
+    return uploadIdMarker;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    ListMultipartUploadsRequest that = (ListMultipartUploadsRequest) o;
+    return Objects.equals(bucket, that.bucket)
+        && Objects.equals(delimiter, that.delimiter)
+        && Objects.equals(encodingType, that.encodingType)
+        && Objects.equals(keyMarker, that.keyMarker)
+        && Objects.equals(maxUploads, that.maxUploads)
+        && Objects.equals(prefix, that.prefix)
+        && Objects.equals(uploadIdMarker, that.uploadIdMarker);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(
+        bucket, delimiter, encodingType, keyMarker, maxUploads, prefix, uploadIdMarker);
+  }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder("ListMultipartUploadsRequest{");
+    sb.append("bucket='").append(bucket).append("'");
+    sb.append(", delimiter='").append(delimiter).append("'");
+    sb.append(", encodingType='").append(encodingType).append(",");
+    sb.append(", keyMarker='").append(keyMarker).append(",");
+    sb.append(", maxUploads=").append(maxUploads);
+    sb.append(", prefix='").append(prefix).append(",");
+    sb.append(", uploadIdMarker='").append(uploadIdMarker).append(",");
+    sb.append('}');
+    return sb.toString();
+  }
```

Comment:
nit: use MoreObjects helper like the other classes do: https://github.com/googleapis/java-storage/blob/763293be11c0cd28717358b54b84db2cfbf4d246/google-cloud-storage/src/main/java/com/google/cloud/storage/multipartupload/model/CompleteMultipartUploadRequest.java#L109-L114